ops/pebble.py

```diff
@@ -3404,6 +3422,32 @@ def replace_identities(self, identities: Mapping[str, IdentityDict | Identity |
 
             body = {'action': 'replace', 'identities': identities_dict}
             self._request('POST', '/v1/identities', body=body)
+            for username, identity in identities_dict.items():
+                if identity is None:
+                    self._security_event(
+                        f'user_updated:{os.getuid()},{username}',
+                        level='INFO',
+                        description=f'Removing identity {username!r} from Pebble.',
+                    )
+                    continue
+                if 'local' in identity:
+                    user_id = identity['local'].get('user-id', 'unknown')
+                else:
+                    user_id = 'unknown'
+                if identity.get('basic') and identity.get('local'):
+                    identity_type = 'basic,local'
+                elif identity.get('basic'):
+                    identity_type = 'basic'
+                elif identity.get('local'):
+                    identity_type = 'local'
+                else:
+                    identity_type = 'unknown'
+                self._security_event(
+                    f'user_added:{os.getuid()},{username},'
+                    f'attributes[{identity["access"]}, {identity_type}, {user_id}]',
```

Comment:
This reads rather awkwardly on two lines (looks like two args). Can we adjust? Maybe build it in a previous statement or similar.